sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -1571,3 +1572,25 @@ case class SetVariable(
   override protected def withNewChildInternal(newChild: LogicalPlan): SetVariable =
     copy(sourceQuery = newChild)
 }
+
+/**
+ * The logical plan for the CALL command.
+ */
+case class Call(procedure: LogicalPlan, args: Seq[Expression]) extends UnaryCommand {
```

Comment:
shall we extend `UnaryRunnableCommand` to avoid creating a corresponding physical node?